pkg/streaming/commands/processor_list.go

```diff
@@ -128,6 +128,14 @@ func (opts *ProcessorListOptions) print(processor *streamv1alpha1.Processor, _ p
 	return []metav1beta1.TableRow{row}, nil
 }
 
+func functionRef(processor *streamv1alpha1.Processor) string {
```

Comment:
personal nit: it's better to attach methods to a receiver so they don't pollute the package. We've had collisions in the past. 

```suggestion
func (*ProcessorListOptions) functionRef(processor *streamv1alpha1.Processor) string {
```